### PR TITLE
150 quiz style what are your interests and whats on your mind

### PR DESCRIPTION
**Pull request by edpe**
https://trello.com/c/QSe0f1dA/150-quiz-style-what-are-your-interests-and-whats-on-your-mind-questions-vv-ee

### CODE REVIEW

**Comment by marxian** [comment]
#### 📝Changed routes:


| [**/quiz/how-are-you-feeling**](https://my-best-life-nj88m6443-neontribe.vercel.app/quiz/how-are-you-feeling) | [**…hat-are-your-interests**](https://my-best-life-nj88m6443-neontribe.vercel.app/quiz/what-are-your-interests) |
|:-:|:-:|
|<a href="https://my-best-life-nj88m6443-neontribe.vercel.app/quiz/how-are-you-feeling">Screenshot of /quiz/how-are-you-feeling</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/UIv02yMdDR8Oz-Rz8TSgGad2Bi4HcGk0H3tSS3ymH5kU1D4S1miby6FHYiArcC5fVzmmh_gsEWFHd6v3mjmRKw.png">(view full size)</a> |<a href="https://my-best-life-nj88m6443-neontribe.vercel.app/quiz/what-are-your-interests">Screenshot of /quiz/what-are-your-interests</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/40oShi1JLyc0LoI8OwSMGDHEKt1HANupA1mYJ_Zstcl1QspS3aZznFPtUiEWAiSO32zA4sOvbriXVNnJv90Ylw.png">(view full size)</a> |



| [**/quiz/whats-on-your-mind**](https://my-best-life-nj88m6443-neontribe.vercel.app/quiz/whats-on-your-mind) |
|:-:|
|<a href="https://my-best-life-nj88m6443-neontribe.vercel.app/quiz/whats-on-your-mind">Screenshot of /quiz/whats-on-your-mind</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/QzSwdTGxkIKgmKGhQ-o_dEWRCwIbMk81BxEjDvxrz9oM6nOE9uyWryyE7tX94PS0fRjbNcgFEqEwt1vSRAcsyw.png">(view full size)</a> |







Commit 920b692e4bca91e9e63d7e3488a0b25e270d9b04 (https://my-best-life-nj88m6443-neontribe.vercel.app).

File: public/img/gaming.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....w3.org/2000/svg" viewBox="0 0 48 48"><path fill="#424242" d="M40,6c0,0,2,8,2,18c0,5.148-0.53,9.765-1.044,13.035C38,37,24,37,24,37l-4,5H8c0,0-2-8-2-18S8,6,8,6 H40z"/><path fill="#bdbdbd" d="M41.11 36C40.58 39.69 40 42 40 42H19c0-3.31 2.69-6 6-6H41.11zM20.12 25H19v-1.8c0-.993.754-1.8 1.68-1.8h3.36c.309 0 .56-.269.56-.6 0-.331-.251-.6-.56-.6h-4.48V19h4.48c.926 0 1.68.807 1.68 1.8 0 .993-.754 1.8-1.68 1.8h-3.36c-.309 0-.56.269-.56.6V25zM27.96 25h-2.8v-1.2h2.8c.309 0 .56-.269.56-.6v-2.4c0-.993.754-1.8 1.68-1.8H33v1.2h-2.8c-.309 0-.56.269-.56.6v2.4C29.64 24.193 28.886 25 27.96 25zM38.273 22.6H34.12v-1.2h4.153c.266 0 .509-.164.575-.39.081-.275-.013-.471-.076-.564C38.666 20.29 38.501 20.2 38.32 20.2h-4.2V19h4.2c.534 0 1.041.276 1.356.738.321.47.408 1.066.24 1.635C39.703 22.095 39.027 22.6 38.273 22.6z"/><path fill="#bdbdbd" d="M38.273,25H34.12v-1.2h4.153c0.266,0,0.509-0.164,0.575-0.39c0.081-0.275-0.013-0.471-0.076-0.564 C38.666,22.69,38.501,22.6,38.32,22.6h-4.2v-1.2h4.2c0.534,0,1.041,0.276,1.356,0.738c0.321,0.47,0.408,1.066,0.24,1.635 C39.703,24.495,39.027,25,38.273,25z"/><g><path fill="#757575" d="M35 38A1 1 0 1 0 35 40 1 1 0 1 0 35 38zM28 38A1 1 0 1 0 28 40 1 1 0 1 0 28 38z"/></g></svg>
```
Comment (Nikomus) [inline public/img/gaming.svg:1]: Minor thing but the PS3 was released in 2006 so we may want something more trendy for the YP 😅

File: public/img/films and tv.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><path fill="#424242" d="M42,37c0,2.762-2.238,5-5,5H11c-2.761,0-5-2.238-5-5V11c0-2.762,2.239-5,5-5h26c2.762,0,5,2.238,5,5	V37z"/><path fill="#b71c1c" d="M31,12h-5v23.5c2.5,0,4.986,0.512,5,0.5S31,12,31,12z"/><path fill="#b71c1c" d="M17,12h5v23.5c-2.5,0-4.986,0.512-5,0.5S17,12,17,12z"/><path fill="#e53935" d="M22,12h-5l9,23.5c2.5,0,4.986,0.512,5,0.5S22,12,22,12z"/></svg>
```
Comment (Nikomus) [inline public/img/films and tv.svg:1]: Are we allowed to use this?

**Review by Nikomus — APPROVED** [review]
Couple of Qs on icons but merge away 🍦

**Pull request merged**